File: React:TyeScript- FE/src/api/models/review.ts
export interface Review {
  id: string;
  filmId: string;
  text: string;
  user: string;
  score: number;
  date: Date;
}

export interface AddReviewModel {
  filmId: string;
  text: string;
  user: string;
  score: number;
}

File: C#-api/DataLayer/Models/Review.cs
namespace films_assignment_api.DataLayer.Models
{
    public class Review : BaseModel
    {
        public string FilmId { get; set; }
        public string Text { get; set; }
        public string User { get; set; }
        public int Score { get; set; }
        public DateTime Date { get; set; }
    }
}
